<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1306</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    background-color:rgb(11, 154, 133);
    border: none;
    border-radius:20px;
    color: white; /* White text */
    padding: 12px 24px; /* Padding */
    text-align: center; /* Center text */
    text-decoration: none; /* Remove underline */
    font-size: 16px; /* Font size */
    cursor: pointer; /* Cursor style */
}

QPushButton:hover {
    background-color:rgb(16, 158, 125) /* Darker green on hover */
}

QPushButton:pressed {
    background-color: #3e8e41; /* Darker green when pressed */
}

QLineEdit{
	border: 2px solid rgb(11, 154, 133);
	border-radius:10px;
}

QComboBox {
    border: 2px solid rgb(44, 154, 112);
    border-radius: 10px;
}
QGroupBox {
    border: 2px solid rgb(44, 154, 112);
    border-radius: 10px;
}
QDateEdit {
    border: 2px solid rgb(44, 154, 112);
    
}

QComboBox::down-arrow,QDateTimeEdit::down-button {
	
	background-color: rgb(255, 255, 255);
    image: url(:/Downloads/arrow.PNG);
    width: 20px; /* Set the width of the arrow */
    height: 20px; /* Set the height of the arrow */
}
QDateTimeEdit::up-button{
	image: url(:/arrow_up.PNG);
	width: 20px; /* Set the width of the arrow */
    height: 20px; /* Set the height of the arrow */
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>1970</x>
      <y>370</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>461</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="title">
     <string>Ajouter un evenement</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Lieu</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Date debut</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>410</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nom">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_lieu">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Prix d'entr√©</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prixentre">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>350</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_description">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Date fin</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_type">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>400</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>Visites d'Ateliers et de Galeries</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vernissages et Expositions </string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ateliers et Cours</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Concours et Prix</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Performances Artistiques</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Festivals </string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>480</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>480</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="datedebutEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>230</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="datefinEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>290</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>480</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ID">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_search">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>80</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Trier par :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>80</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>ID ascendant</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ID descendant</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nom ascendant</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nom descendant</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Description ascendant</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Description descendant</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lieu ascendant</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lieu descendant</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Date debut ascendant</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Date debut descendant</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Date fin ascendant</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Date fin descendant</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Prix d'entr√© ascendant</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Prix d'entr√© descendant</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Type ascendant</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Type descendant</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>80</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Filtrer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>P√©riode :</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>20</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>140</y>
      <width>801</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>70</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>G√©nerer une copie PDF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_refresh">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>490</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Recherche</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1306</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
